[DO NOT MERGE] Adding DataStore JS lazy loading docs (#4786)
aws-amplify/docs: src/fragments/lib/datastore/js/data-access/query-predicate-multiple-snippet.mdx

> When using multiple conditions, there is an implicit `and` defined to mirror the [GraphQL Transformer condition statements](/cli/graphql/data-modeling). For example with multiple conditions:

Author: ```suggestion
When using multiple conditions, you can wrap the predicates with the `and` operator. For example with multiple conditions:
```
